src/Assets/ugui-mvvm/CommandBinding.cs

```diff
@@ -54,7 +54,7 @@ public object Parameter
         INPCBinding.PropertyPath _vmProp;
         private ICommand _command;
 
-        public INPCBinding.ComponentPath ViewModel
+        public INPCBinding.ComponentPath Source
```

Comment:
Can you actually add [Deprecated] properties for the original property names?  It would lessen problems for consumers of the library.